pkg/token/generator/generator_types.go

```diff
@@ -26,9 +26,13 @@ type CodeGenerateInfo struct {
 }
 
 type AuthorizationCodeGenerator interface {
-	GenCode(info *CodeGenerateInfo) string
+	Generate(info *CodeGenerateInfo) string
```

Comment:
Why we need three interface, but exactly the same?

Reply:
fixed